configure.ac

```diff
@@ -719,6 +720,31 @@ then
                                   [#include <freerdp/utils/stream.h>])])
 fi
 
+# Find location of Stream_New and Stream_free
+if test "x${have_freerdp}" = "xyes" -a "x${have_winpr}" = "xyes"
+then
+    AC_CHECK_LIB([winpr], [Stream_New, Stream_Free],
+                 [WINPR_LIBS="$WINPR_LIBS -lwinpr"],
+                 [AC_CHECK_LIB([winpr-utils], [Stream_New, Stream_Free],
+                               [WINPR_LIBS="$WINPR_LIBS -lwinpr-utils"],
+                               [AC_MSG_WARN([
+  ------------------------------------------
+  Unable to locate stream functions in winpr
+  libraries.  RDP will be disabled.
+  ------------------------------------------])
+                                    have_freerdp=no])])
+fi
+
+if test "x${have_freerdp}" = "xyes"
+then
+    AC_CHECK_LIB([freerdp-client], [freerdp_channels_new],
```

Comment:
I think this might be copypasta of one of the existing checks:

https://github.com/apache/guacamole-server/blob/7024b290abc2744168bd93de9ceb781f62959e60/configure.ac#L561-L570

Reply:
Haha - oops.  I think I may have copied it down for referenced.  Removed.